pkgs/development/libraries/openal-soft/default.nix

```diff
@@ -2,6 +2,7 @@
 , alsaSupport ? !stdenv.isDarwin, alsa-lib
 , pulseSupport ? !stdenv.isDarwin, libpulseaudio
 , CoreServices, AudioUnit, AudioToolbox
+, glibc
```

Comment:
this will fail on musl-based platforms.
Suggestion:
```diff
diff --git a/pkgs/development/libraries/openal-soft/default.nix b/pkgs/development/libraries/openal-soft/default.nix
index 6e1026529bd..d2cdebdf355 100644
--- a/pkgs/development/libraries/openal-soft/default.nix
+++ b/pkgs/development/libraries/openal-soft/default.nix
@@ -2,7 +2,6 @@
 , alsaSupport ? !stdenv.isDarwin, alsa-lib
 , pulseSupport ? !stdenv.isDarwin, libpulseaudio
 , CoreServices, AudioUnit, AudioToolbox
-, glibc
 }:
 
 stdenv.mkDerivation rec {
@@ -28,7 +27,7 @@ stdenv.mkDerivation rec {
 
   nativeBuildInputs = [ cmake ];
 
-  buildInputs = lib.optional (stdenv.buildPlatform != stdenv.hostPlatform) glibc
+  buildInputs = lib.optional (stdenv.buildPlatform != stdenv.hostPlatform) stdenv.cc.libc
     ++ lib.optional alsaSupport alsa-lib
     ++ lib.optional pulseSupport libpulseaudio
     ++ lib.optionals stdenv.isDarwin [ CoreServices AudioUnit AudioToolbox ];
```

Reply:
Applied.